src/components/MeetTheTeam/team.js

```diff
@@ -0,0 +1,44 @@
+import fadime from "./images/fadime.png";
+import halit from "./images/halit.png";
+import khaldoon from "./images/khaldoon.png";
+import ahmet from "./images/ahmet.png";
+import zeynep from "./images/zeynep.png";
+
+const team = [
+  {
+    avatar: halit,
+    name: "HALIT BATUR",
+    title: "meetTheTeam.titles.teamLead",
+    github: "https://github.com/halitbatur",
```

Comment:
you can fetch them directly from i18n

Reply:
> you can fetch them directly from i18n

github is the same in every language, that's why no need to translate I guess